codeGeneration.test.js

```diff
@@ -60,15 +60,15 @@ describe("@font-face", () => {
 
     expect(fontFace).toEqual(stripIndent`
       @font-face {
-          font-family: "My font";
+          font-family: "my-font";
           src: url("./test/__fixtures__/Fraunces-VF.ttf")
       }
     `);
   });
 
   test("includes font-weight declaration if wght axis present", () => {
     const fontData = {
-      name: "My font",
```

Comment:
Just curious, how is this not safe when it's quoted?

Reply:
This is a temporary patch. We're getting the name from two different places, which might break specimens. This PR uses the selector which is "universal" so at least specimens can be built properly now.

Created https://github.com/kabisa/specimen-skeleton-support/issues/18 to investigate further.

Reply:
Hold on, I will address #18 in a different PR, as that is a much better solution. This PR merely fights the symptom, not the cause. Closing it.